### ISSUE TITLE
options.blocks passed to remark-parser is in an invalid format

### ISSUE THREAD

**Issue by peterschriever** [probablyup/markdown-to-jsx#109]
**Update below, tldr; my webpack config did not properly load json files.**

Hello there, I just installed markdown-to-jsx version 5.3.3 and remark-parse 3.0.1 but I am encountering the following trying to display Markup content:
```
Uncaught Error: Invalid value `[
  "address",
  "article",
  "aside",
  "base",
  "basefont",
  "blockquote",
  "body",
  "caption",
  "center",
  "col",
  "colgroup",
  "dd",
  "details",
  "dialog",
  "dir",
  "div",
  "dl",
  "dt",
  "fieldset",
  "figcaption",
  "figure",
  "footer",
  "form",
  "frame",
  "frameset",
  "h1",
  "h2",
  "h3",
  "h4",
  "h5",
  "h6",
  "head",
  "header",
  "hgroup",
  "hr",
  "html",
  "iframe",
  "legend",
  "li",
  "link",
  "main",
  "menu",
  "menuitem",
  "meta",
  "nav",
  "noframes",
  "ol",
  "optgroup",
  "option",
  "p",
  "param",
  "pre",
  "section",
  "source",
  "title",
  "summary",
  "table",
  "tbody",
  "td",
  "tfoot",
  "th",
  "thead",
  "title",
  "tr",
  "track",
  "ul"
]
` for setting `options.blocks`
    at Of.setOptions (bundle.js:35588)
    at Of.Parser (bundle.js:33996)
    at new Of (bundle.js:33909)
    at Function.parse (bundle.js:32745)
    at compiler (bundle.js:31477)
    at Component (bundle.js:31518)
    at bundle.js:16674
    at measureLifeCyclePerf (bundle.js:16444)
    at ReactCompositeComponentWrapper._constructComponentWithoutOwner (bundle.js:16673)
    at ReactCompositeComponentWrapper._constructComponent (bundle.js:16648)
```
I've tried simplifiying my markdown using component as much as possible and as close to the provided examples as possible. My code currently looks like this:
```
import React, { Component } from 'react'
import Markdown from 'markdown-to-jsx'

export default class MarkdownContentArea extends Component {
  render() {
    content = `
      # Hello world!
    `.trim();
    return (
      <Markdown>
        {content}
      </Markdown>
    )
  }
}
```

After reading through the error a bit more carefully and reading through the remark-parse code a bit I've found the following check in their package [(github link to the file)](https://github.com/wooorm/remark/blob/master/packages/remark-parse/lib/set-options.js#L42-L50):
```
    if (
      (key !== 'blocks' && typeof value !== 'boolean') ||
      (key === 'blocks' && typeof value !== 'object')
    ) {
      throw new Error(
        'Invalid value `' + value + '` ' +
        'for setting `options.' + key + '`'
      );
    }
```
Which indicates that something might be going on with the passing of options.blocks being an array instead of an object. However I couldnt quite figure out why this is, or why I am the first to be having this problem since it doesn't appear like the file has changed much.

**Comment by peterschriever** [comment]
Update: I just realised the options.blocks variable was being might have been retrieved from a default file in the remark-parse package. This is probably true because there is a file called block-elements.json in their package libs which contains the exact array output. I figured that my webpack config might have been wonky with json files, since I also had some errors right after installing the markdown-to-jsx package. I suppose my biggest fault might be that I'm still using webpack v1.x, haha.. After installing the [webpack-contrib/json-loader](https://github.com/webpack-contrib/json-loader) package everything started working properly.
